release/models/bgp/openconfig-bgp-policy.yang

```diff
@@ -578,17 +585,26 @@ module openconfig-bgp-policy {
     leaf-list community-member {
       type union {
         type oc-bgp-types:bgp-std-community-type;
-        type oc-bgp-types:bgp-community-regexp-type;
+        type oc-types:posix-eregexp;
         type oc-bgp-types:bgp-well-known-community-type;
       }
       description
-        "members of the community set.
-        For an ADD operation these are the communities that will be added;
-        the regexp type is not valid in this operation.
+        "Members of the community set.
+        For an ADD operation these are the communities that will be
+        added.  The regexp type is not valid in this operation.
+
         For REMOVE or REPLACE operations then matching communities will
         be removed unless match-set-options is INVERT which will
         reverse this to mean that anything that does not match will be
-        removed.";
+        removed.
+
+        For MATCH operations the posix-eregexp type should be evaluated
+        against each community associated with a prefix one community
+        at a time.  Communities must be represented as strings in formats
+        conforming to oc-bgp-types:bgp-std-community-type.  For example:
+        - `1000:1000` for a standard community
+        - `route-origin:1000:1000` for the origin type extended community,
```

Comment:
Should this example be in the extended community description?

Alternatively should this regex format excerpt be documented in one place, like a `bgp-community-eregexp` (or separate `std`/`ext`/`large`) type?

It might be important to be able to document the exact format of all possible extended community values.

Reply:
Thanks for this.  I moved the extended community example to `leaf-list ext-community-member`.

Regarding example extended community values, one could say that [oc-bgp-types:bgp-ext-community-type](https://github.com/openconfig/public/blob/5d24889a323a84fd077cbb83c2c7b5fb4896a6fe/release/models/bgp/openconfig-bgp-types.yang#L441-L643) does this.   Take a look there and see if you agree that it is satisfactory.

Reply:
I think that covers it well, though the main description does not cover `link-bandwidth:`. Perhaps this change could update that description?

Reply:
Done!